com.hazelcast.jet.kafka.impl.StreamKafkaP_TimestampModesTest.test_sourceKafka_withNativeTimestamps

http://jenkins.hazelcast.com/job/jet-oss-pr-builder/168/testReport/junit/com.hazelcast.jet.kafka.impl/StreamKafkaP_TimestampModesTest/test_sourceKafka_withNativeTimestamps/

```
Error Message
expected:<[1, 2]> but was:<[2]>
Stacktrace
java.lang.AssertionError: expected:<[1, 2]> but was:<[2]>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.hazelcast.jet.pipeline.StreamSourceStageTestBase.lambda$test$3(StreamSourceStageTestBase.java:115)
	at com.hazelcast.jet.core.JetTestSupport$1.run(JetTestSupport.java:196)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1302)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1319)
	at com.hazelcast.jet.core.JetTestSupport.assertTrueEventually(JetTestSupport.java:161)
	at com.hazelcast.jet.pipeline.StreamSourceStageTestBase.test(StreamSourceStageTestBase.java:116)
	at com.hazelcast.jet.kafka.impl.StreamKafkaP_TimestampModesTest.test_sourceKafka_withNativeTimestamps(StreamKafkaP_TimestampModesTest.java:79)
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamSourceStageTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamSourceStageTest.java
@@ -39,7 +39,7 @@ public class StreamSourceStageTest extends StreamSourceStageTestBase {
 
     @BeforeClass
     public static void beforeClass1() {
-        IMap<Integer, Integer> sourceMap = instances[0].getMap(JOURNALED_MAP_NAME);
+        IMap<Integer, Integer> sourceMap = instance.getMap(JOURNALED_MAP_NAME);
         sourceMap.put(1, 1);
         sourceMap.put(2, 2);
     }
@@ -122,14 +122,14 @@ public class StreamSourceStageTest extends StreamSourceStageTestBase {
 
     @Test
     public void test_withTimestampsButTimestampsNotUsed() {
-        IList sinkList = instances[0].getList("sinkList");
+        IList sinkList = instance.getList("sinkList");
 
         Pipeline p = Pipeline.create();
         p.drawFrom(createSourceJournal())
          .withIngestionTimestamps()
          .drainTo(Sinks.list(sinkList));
 
-        instances[0].newJob(p);
+        instance.newJob(p);
         assertTrueEventually(() -> assertEquals(Arrays.asList(1, 2), new ArrayList<>(sinkList)), 5);
     }
 

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamSourceStageTestBase.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamSourceStageTestBase.java
@@ -49,7 +49,7 @@ import static org.junit.Assert.fail;
 
 public abstract class StreamSourceStageTestBase extends JetTestSupport {
 
-    protected static JetInstance[] instances;
+    protected static JetInstance instance;
     static final String JOURNALED_MAP_NAME = "journaledMap";
     private static JetTestInstanceFactory factory = new JetTestInstanceFactory();
 
@@ -76,7 +76,7 @@ public abstract class StreamSourceStageTestBase extends JetTestSupport {
                 .setEnabled(true));
         // use 1 partition for the map journal to have an item in each ption
         config.getHazelcastConfig().setProperty(PARTITION_COUNT.getName(), "1");
-        instances = factory.newMembers(config, 2);
+        instance = factory.newMember(config);
     }
 
     @AfterClass
@@ -103,12 +103,13 @@ public abstract class StreamSourceStageTestBase extends JetTestSupport {
             return;
         }
         stageWithTimestamps
+                .setLocalParallelism(1)
                 .peek()
                 .window(WindowDefinition.tumbling(1))
                 .aggregate(counting())
                 .peek()
                 .drainTo(Sinks.fromProcessor("s", ProcessorMetaSupplier.of(WatermarkLogger::new)));
-        Job job = instances[0].newJob(p);
+        Job job = instance.newJob(p);
 
         HashSet<Long> expectedWmsSet = new HashSet<>(expectedWms);
 

